dftimewolf/lib/processors/turbinia_base.py

```diff
@@ -174,6 +175,48 @@ def _ExtractFiles(self, tgz_path: str, path_to_collect: str) -> str:
     local_path = os.path.join(tempdir, path_to_collect.lstrip('/'))
     return local_path
 
+  def UploadEvidence(self, file_path: Path) -> Optional[str]:
+    """Uploads files to Turbinia via the API server.
+    
+    Args:
+      file_path: Path to the file to be uploaded.
```

Comment:
Is it a compressed file that the API server is expecting? or will any file do?

Reply:
The method itself can upload any type of file. The API doesn't expect a specific type. Although dfTimewolf uses it to generate a CompressedDirectory request, the ```UploadEvidence``` method could be used for any file type.